pkgs/applications/networking/mailreaders/trojita/default.nix

```diff
@@ -27,6 +28,13 @@ mkDerivation rec {
     sha256 = "0xpxq5bzqaa68lkz90wima5q2m0mdcn0rvnigb66lylb4n20mnql";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
please also set `name = "CVE-2020-15047.patch";`, so our tooling can understand this fixes the CVE.